sample-app/src/main/kotlin/example/com/androidxsample/view/MainActivity.kt

```diff
@@ -11,6 +11,22 @@ import moxy.presenter.ProvidePresenter
 
 class MainActivity : MvpAppCompatActivity(R.layout.activity_main), MainView {
 
+    override fun tryTag(tag: String) {
+
+    }
+
+    override fun tryView(view: String) {
+
+    }
+
+    override fun tryViewCommands(viewCommands: String) {
+
+    }
+
+    override fun tryViews(views: String) {
+
+    }
+
```

Comment:
Why do we need these methods in sample app. I don't think it belongs here, but rather in tests in `ParameterNameClashView`.